### PR TITLE
Enhance node operator documentation

### PR DESCRIPTION
**Pull request by candida-d**

### CODE REVIEW

File: docs/node-operators/requirements/infrastructure.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+| CPU | 4 cores at >3Ghz | 6 cores at >= 4Ghz|
+| RAM   | 8GB        | 16GB        |
+| Storage   | 256GB SSD| 1TB NVMe SSD |
+
+The impact of using fewer cores than recommended is that the critical parts (data node and non-validators) will be starved of CPU resources, which will reduce the maximum throughput of the network.
+
+The impact of having slower cores than recommended (or older cores that have a reduced IPC rate compared to modern Zen3 cores) is that the maximum throughput will be reduced due to the bottlenecks being single threaded.
+
+## Token requirements
+To run a validator node, the node operator must have a minimum amount of Vega available to stake to that node: <NetworkParameter frontMatter={frontMatter} param="reward.staking.delegation.minimumValidatorStake" hideName={false} suffix="tokens" />
```
Comment (candida-d) [inline docs/node-operators/requirements/infrastructure.md:28]:
```suggestion
To run a validator node, the node operator must have a minimum amount of Vega available to stake to that node: <NetworkParameter frontMatter={frontMatter} param="reward.staking.delegation.minimumValidatorStake" hideName={false} formatter="governanceToken" suffix="tokens" />
```

**Comment by edd** [comment]
- Following the sandbox instructions, I struggled to find the Sepolia Vega contract address to faucet myself some Vega (after finding a Sepolia faucet). https://docs.vega.xyz/testnet/tutorials/staking-tokens#ethereum-addresses has the contracts - might be worth linking there? Or using the `EthAdresses` table.

**Comment by edd** [comment]
https://deploy-preview-189--vega-docusaurus.netlify.app/docs/testnet/node-operators/get-started/setup-validator#set-up-the-vega-wallet

The nodewallet import instructions should be clearer that wallet-path is actually the path to the file, not just the wallet ome:

`--wallet-path="/home/validator/.local/share/vega/wallets/validator.3cdcbfd7.isolated"`

File: docs/node-operators/get-started/setup-validator.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+Then ensure the `max_packet_msg_payload_size` is at least 16384.
+
+Under `Mempool Configuration Option`, ensure that `broadcast = true`.
+
+### Save Tendermint genesis
+To start successfully, tendermint needs the genesis file from the network you will be trying to join. This file need to be located in `YOUR_TENDERMINT_HOME/config/genesis.json`. 
+
+You can find genesis files for network supported by the community in the [networks repository](https://github.com/vegaprotocol/networks). 
+
```
Comment (peterbarrow) [inline docs/node-operators/get-started/setup-validator.md:157]: Add network-internal here too

**Review by peterbarrow — APPROVED** [review]

**Pull request merged**